crown-court-proceeding/build.gradle

```diff
@@ -46,6 +50,12 @@ dependencies {
     implementation "io.sentry:sentry-logback"
     implementation "io.sentry:sentry-spring-boot-starter"
 
+    implementation "org.springframework.cloud:spring-cloud-starter-aws:$versions.springFrameworkCloudVersion"
+    implementation "org.springframework.cloud:spring-cloud-starter-aws-messaging:$versions.springFrameworkCloudVersion"
+    implementation "org.springframework.cloud:spring-cloud-aws-autoconfigure:$versions.springFrameworkCloudVersion"
```

Comment:
As per the [release notes](https://github.com/awspring/spring-cloud-aws): Spring Cloud AWS 3.0.x is compatible with Spring Boot 3.0.x. As per the [Spring Guide](https://docs.awspring.io/spring-cloud-aws/docs/3.0.0/reference/html/index.html#sqs-integration) we can use the spring-cloud-aws-starter-sqs

Reply:
Sri, As you can see, that is what I've used and tested and it is working fine. Not sure if I'm missing anything here

I thought I clarified everything in our call. I  don't see a point in delaying this as this is our last day of Sprint

If you till have doubts, I'm happy to get on a call and discuss to close this. Back and forth comments is not going to help

Reply:
As discussed, this will be addressed as part of
https://dsdmoj.atlassian.net/browse/LCAM-833